configs/example_configs_validation.py

```diff
@@ -17,7 +17,7 @@ def main():
             validate_fragment(
                 "envoy.config.bootstrap.v3.Bootstrap",
                 yaml.safe_load(pathlib.Path(arg).read_text()))
-        except (ParseError, KeyError) as e:
+        except (ParseError, yaml.parser.ParserError, KeyError) as e:
```

Comment:
this is a fix to the config_validation to prevent it borking on yaml parser errors